content/docs/1.61/deployment.md

```diff
@@ -283,7 +283,7 @@ docker run \
 * [Badger file permissions as non-root service](https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/badger/docs/storage-file-non-root-permission.md)
 
 ### Cassandra
-* Supported versions: 3.4+
+* Supported versions: 4.0+
```

Comment:
please also add to content/docs/next-release/deployment.md

Reply:
also, I don't think it should say +, but explicitly 4.x and 5.x

Reply:
done